README.md

```diff
@@ -10,7 +10,7 @@ expressive.
   particles, following relatively few yet composable grammar rules.
 - Regular: particles are organized into families, each with a unique prefix, enabling
   quick recognition and accelerating learning.
-- Expressive: predicates can be combined in a variety of ways to add more details, greatly
+- Expressive: predicates can be combined and transformed in a variety of ways, greatly
```

Comment:
If that's better, I'll update the other two places (book and main page)

Reply:
Indeed !